release-notes/changelog.adoc

```diff
@@ -1,27 +1,90 @@
 # Changelog
 
+## v0.24.0
+
+Released on: Tue Dec 3 15:34:34 2019 +0100
+
+- CI: Support for Kubernetes 1.15
+- CI: Support for Kubernetes 1.16
+- Add and fix a couple of test cases
+- Support for pause and unpausing VMs
+- Update of libvirt to 5.6.0
+- Fix bug related to parallel scraping of Prometheus endpoints
+- Fix to reliably test VNC
+
+## v0.23.0
+
+Released on: Mon Nov 4 16:42:54 2019 +0100
+
+- Guest OS Information is available under the VMI status now
+- Updated to Go 1.12.8 and latest bazel
+- Updated go-yaml to v2.2.4, which has a ddos vulnerability fixed
+- Cleaned up and fixed CRD scheme registration
+- Several bugfixes
+- Many CI improvements (e.g. more logs in case of test failures)
+
+## v0.22.0
+
+Released on: Thu Oct 10 18:55:08 2019 +0200
+
+- Support for Nvidia GPUs and vGPUs exposed by Nvidia Kubevirt Device Plugin.
+- VMIs now successfully start if they get a 0xfe prefixed MAC address assigned from the pod network
+- Removed dependency on host semanage in SELinux Permissive mode
+- Some changes as result of entering the CNCF sandbox (DCO check, FOSSA check, best practice badge)
+- Many bug fixes and improvements in several areas
+- CI: Introduced a OKD 4 test lane
+- CI: Many improved tests, resulting in less flakyness
+
+## v0.21.0
+
+Released on: Mon Sep 9 09:59:08 2019 +0200
+
+- CI: Support for Kubernetes 1.14
+- Many bug fixes in several areas
+- Support for `virtctl migrate`
+- Support configurable number of controller threads
+- Support to opt-out of bridge binding for podnetwork
+- Support for OpenShift Prometheus monitoring
+- Support for setting more SMBIOS fields
+- Improved containerDisk memory usage and speed
+- Fix CRI-O memory limit
+- Drop spc_t from launcher
+- Add feature gates to security sensitive features
+
 ## v0.20.0
 
 Released on: Fri Aug 9 16:42:41 2019 +0200
 
 - Containerdisks are now secure and they are not copied anymore on every start.
+Old containerdisks can still be used in the same secure way, but new
```

Comment:
I think the indentation is not correct here. Did you notice it?

Reply:
This is due to missing hyphen in the source.
The line starts with "Old containerdisks..." is without '-' at the beginning.
As you can see in output of:
`git -C kubevirt show v0.20.0`